server/lib/applications.js

```diff
@@ -40,6 +40,34 @@ export const saveApplication = (id, body, storage) => new Promise((resolve, reje
     .catch(reject);
 });
 
+/*
+ * Move application to rearrange order.
+ */
+export const moveApplication = (id, direction, storage) =>
```

Comment:
need to add some tests for this function

Reply:
Done.